### DISCUSSION TITLE
git unable to access: Error -50 setting GnuTLS cipher list starting with +VERS-TLS1.3:+SRP

### DISCUSSION THREAD

**Discussion by sudoyang** [community/community#22266] [New to GitHub]
<p>Hi guys, I just find my git is broken with the following issue. Basically it is dead when I run “git clone xxxx”, which gives me error below. I am running Ubuntu 19.04 beta currently. git version is 2.20.1 Does anyone know how I can fix this? Thanks! $git clone <a href="http://github.com/hotoo/pangu.vim.git" rel="nofollow noopener">http://github.com/hotoo/pangu.vim.git</a> Cloning into ‘pangu.vim’… fatal: unable to access ‘<a href="http://github.com/hotoo/pangu.vim.git/" rel="nofollow noopener">http://github.com/hotoo/pangu.vim.git/</a>’: Error -50 setting GnuTLS cipher list starting with +VERS-TLS1.3:+SRP</p>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/sudoyang">@sudoyang</a>,</p>
<p>This is not a problem with git or github, but with your local configuration. You’ve set GIT_SSL_CIPHER_LIST or the http.sslCipherlList to something your currently used version of gnutls does not support. When git then tells libcurl to tell gnutls to use this cipher list, gnutls tells curl that it can’t do this, and curl feeds this back to git, which is the error you are seeing.</p>

**Comment by sudoyang** [comment]
<p>Thanks! I have recompiled git against OpenSSL.</p>
